static/js/src/cve/cve.js

```diff
@@ -148,19 +87,36 @@ function handleFilters() {
 }
 handleFilters();
 
-// Removes a parameter from the filterParams array
+// Removes a parameter from URLSearchParams object
 function removeParam(param, value) {
-  if (param === "version") {
-    let statusIndex = filterParams
-      .map((param) => param.param)
-      .indexOf("status");
-    filterParams.splice(statusIndex, 1);
+  if (value === "maintained") {
+    maintainedReleases.forEach(function (release) {
+      urlParams.delete(param, release);
+    });
```

Comment:
We can remove this catch if all the checkboxes are already unticked before submit.